### ISSUE TITLE
Failed to build syntax

### ISSUE THREAD

**Issue by ivance00** [dschwen/autocomplete-moose#1]
Updated Marmot and Moose submodule, now I get an error message in Atom "Failed to build Moose Syntax".  Error occurs when editing files from any location including moose, marmot-problems, and my own folder.  Reinstalled Atom as well as the Moose packages and the problem persists.  

![screen shot 2015-09-02 at 1 18 20 pm](https://cloud.githubusercontent.com/assets/13019972/9640071/3e2f63cc-5175-11e5-9299-5edc5f0ecb1c.png)

**Comment by dschwen** [comment]
Can you open the developer console of atom and see if you can find the
actual error message? Are you starting atom from the command line? Starting
from the dock or relying on serfdom restore will cause atom to see an
incomplete environment. I.e. it won't see all the paths and environment
variables it needs to build and run moose.
Also try installing the make-runner package and check if you can build
moose from inside atom.

**Comment by ivance00** [comment]
I tried to open the developer console and attached a screenshot.    It says
"unhandled promise rejection", I am not sure if this is what you were
trying to find out.  I usually start atom from the command line, but not
every time.  The problem is there for either occasion.  I installed the
make-runner and it says make succeeded.  I copied the full path for the
input file I am currently working
on; /Users/meegenrc/projects/marmot/ian/newtest.i
At this point I have no idea why it stopped working, or if I did anything
to cause it, but I appreciate your help

**Comment by dschwen** [comment]
@ivance00 I don't see the screenshot. It should say a bit more than just _unhandled promise rejection_. If you are working on a mac (per your first screenshot) you need to _completely_ close Atom (i.e. Using Quit in the File menu) - just closing the window is _not_ sufficient. Then in your terminal make sure you can run `./modules-opt --syntax` or any MOOSE executable without an error. If that works then start Atom using `atom .` in that _same_ terminal. This must work!

**Comment by ivance00** [comment]
![screen shot 2015-09-03 at 2 07 02 pm](https://cloud.githubusercontent.com/assets/13019972/9740426/4c7672e6-561a-11e5-849b-008560f5a81f.png)
The error says, "Unhanded promise rejection-->Promise with error:-->Object".  I have never used the developer console before so I am not sure what to do in that regard.  
I ran an executable with --syntax, and the next input file I opened it said "Rebuilding Moose Syntax". It stayed blue this time; no error.  Thank you for your help
